.circleci/config.yml

```diff
@@ -143,6 +191,9 @@ workflows:
   version: 2
   build-test-deploy:
     jobs:
+      - run_tests:
+          <<: *feature_branch
```

Comment:
Could this run for both the main branch as well as feature branches?

Reply:
Updated the config so that it should now run on all branches

Reply:
And yes going back to your first comment.  The profile switching is required to install Drupal on circleci (it's explained in the new profiles README file).